### ISSUE TITLE
blur behind background

### ISSUE THREAD

**Issue by parliament718** [sorich87/bootstrap-tour#402]
Is there any way to make the backdrop overlay have the effect of blurring the content behind it, rather than darkening it?

I've look at some possible solutions such as CSS3 ```filter: blur(20px)``` but it seems that would need to be applied to the content not the overlay to work.

Is there a simple way to get this effect that I'm overlooking?

**Comment by sorich87** [comment]
Yes, it needs to be applied to the content. What you can do it to use the onShow callback to apply it.

A solution we can consider in the future is to add a class to the page body when the backdrop is applied so that it's easier to apply custom CSS in that case without using onShown (actually, if you submit a pull request for that, I'll be happy to merge it).
